chain/substreams/src/lib.rs

```diff
@@ -1,8 +1,53 @@
 mod chain;
-mod codec;
+pub mod codec;
 mod data_source;
+pub mod mapper;
 mod trigger;
-
+use crate::codec::EntitiesChanges;
 pub use chain::*;
 pub use data_source::*;
+use graph::blockchain;
+use graph::blockchain::{BlockHash, BlockPtr};
+use graph::prelude::BlockNumber;
 pub use trigger::*;
+
+#[derive(Clone, Debug, Default)]
+pub struct Block {
```

Comment:
Can you please leave this in chain.rs ?

Reply:
Yes, moving it in `chain.rs` like it's done on master